tools/test-case-generators/distro-arch-imagetype-map.json

```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< HEAD
```

Comment:
Leftover conflict markers.

Reply:
Result of the gazillion rebases I had to do 😭  Thanks!